src/sections/pages/users/index.js

```diff
@@ -165,11 +222,11 @@ export default class Users extends Component {
                             <h1 className="react-rainbow-admin-users_card-title">Total users</h1>
                             <div className="react-rainbow-admin-users_chart">
                                 <Chart
-                                    labels={['January', 'February', 'March', 'April', 'May', 'Jun', '']}
+                                    labels={['January', 'February', 'March', 'April', 'May', 'Jun', 'Jul', '']}
                                     type="line"
                                     showLegend={false}
                                     maintainAspectRatio={false}>
-                                    <Dataset values={[370, 90, 950, 530, 800, 960, 650]} borderColor="#1de9b6" />
+                                    <Dataset values={[370, 90, 950, 530, 800, 960, 650, 1000]} borderColor="#1de9b
```

Comment:
fix the data passed to charts:
if label has 7 items then dataset values must have 7 items
don't use an empty string to complete only put the same number of items in both

and here there is the problem because ci is failing, you need to close the component ```/>```